src/semantic.less

```diff
@@ -13,63 +13,167 @@
 // stylelint-disable no-invalid-position-at-import-rule
 
 /* Global */
-& { @import 'definitions/globals/reset'; }
-& { @import 'definitions/globals/site'; }
+& {
```

Comment:
not desired, see https://github.com/fomantic/Fomantic-UI/pull/2612#discussion_r1050598585

Reply:
fixed - file is ignored by Prettier